src/components/SchemaInput.vue

```diff
@@ -102,7 +111,7 @@
     if (inputType.value === 'json') {
       emit('update:modelValue', mapper.map('SchemaValuesResponse', { values: record.value, schema: props.schema }, 'SchemaValues'))
```

Comment:
Since these are linked, we could probably avoid this check altogether and emit the model value update in the computed setter if we want

Reply:
+1 to just using the setter for this